### PR TITLE
docs: API promotion documentaiton

### PR DESCRIPTION
**Pull request by ytvnr**
issue
https://github.com/gravitee-io/issues/issues/5921

### CODE REVIEW

**Review by phiz71 — APPROVED** [review]

File: pages/apim/3.x/user-guide/publisher/promote-api.adoc [hunk 1]
```diff
@@ -11,27 +11,30 @@ The following sections describe how you can promote an API from one environment
...
+** Another one linked to Dev and QA environments
+** An API to promote
 +
 image::apim/3.x/api-publisher-guide/promote-apis/graviteeio-promote-api-cockpit-graph.png[Map of installations in Cockpit]
-+
-
-NOTE: You may only promote an API from one environment to another [underline]#*from within an organization*# (you are not permitted to promote APIs from one organization to another). 
 
-== Request Promotion or an API
+== Request promotion for an API
```
Comment (tiffanymarchi) [inline pages/apim/3.x/user-guide/publisher/promote-api.adoc:27]:
```suggestion
== Request Promotion For An API
```

File: pages/apim/3.x/user-guide/publisher/promote-api.adoc [hunk 2]
```diff
@@ -11,27 +11,30 @@ The following sections describe how you can promote an API from one environment
...
-The following actions must be done with an API Publisher on `DEV` environment.
 
+You only need to have *API-DEFINITION* link:/apim/3.x/apim_adminguide_roles_and_permissions.html[permissions^] to request a promotion.
+
+Promotion requests are logged in link:/apim/3.x/apim_publisherguide_audit.html[Audits trail^].
+
+The following actions must be done with an API Publisher on `DEV` environment.
 
-. Go to the API to promote and click on image:icons/promote-button.png[role="icon"] *PROMOTE* button
+. Go to the API to promote and click on image:icons/promote-button.png[role="icon"] button
```
Comment (tiffanymarchi) [inline pages/apim/3.x/user-guide/publisher/promote-api.adoc:37]:
```suggestion
. Go to the API to promote and click the image:icons/promote-button.png[role="icon"] button.
```

File: pages/apim/3.x/user-guide/publisher/promote-api.adoc [hunk 3]
```diff
@@ -51,6 +54,8 @@ NOTE: If a promotion has already been accepted for an API, you can request a new
 
 == Accepting a Promotion
```
Comment (tiffanymarchi) [inline pages/apim/3.x/user-guide/publisher/promote-api.adoc:55]:
```suggestion
== Accepting A Promotion
```

File: pages/apim/3.x/user-guide/publisher/promote-api.adoc [hunk 4]
```diff
@@ -51,6 +54,8 @@ NOTE: If a promotion has already been accepted for an API, you can request a new
...
 
+You need to have *ENVIRONMENT-API* link:/apim/3.x/apim_adminguide_roles_and_permissions.html[permissions^] to be able to accept or reject a promotion request.
```
Comment (tiffanymarchi) [inline pages/apim/3.x/user-guide/publisher/promote-api.adoc:57]:
```suggestion
To be able to accept or reject a promotion request, you need to have *ENVIRONMENT-API* link:/apim/3.x/apim_adminguide_roles_and_permissions.html[permissions^].
```

File: pages/apim/3.x/user-guide/publisher/promote-api.adoc [hunk 5]
```diff
@@ -51,6 +54,8 @@ NOTE: If a promotion has already been accepted for an API, you can request a new
...
+
 As a user of `QA` environment, promotion requests are visible in the *Tasks* section.
 
 . Go to the *Tasks* section
```
Comment (tiffanymarchi) [inline pages/apim/3.x/user-guide/publisher/promote-api.adoc:61]:
```suggestion
. Go to the *Tasks* section.
```

File: pages/apim/3.x/user-guide/publisher/promote-api.adoc [hunk 6]
```diff
@@ -62,15 +67,3 @@ image::apim/3.x/api-publisher-guide/promote-apis/graviteeio-promote-api-promote-
 ** Accepting the promotion will:
 *** Create or update the API (depending on if it has already been promoted or not)
```
Comment (tiffanymarchi) [inline pages/apim/3.x/user-guide/publisher/promote-api.adoc:68]:
```suggestion
*** Create or update the API (depending on if it has already been promoted or not).
```

**Review by tiffanymarchi — COMMENTED** [review]
Hi @ytvnr -- Thanks for your help on this! Everything looks pretty great. I will have to make a couple very minor edits and will do that after you're done with this. 

Thanks again!

**Pull request merged**